### DISCUSSION TITLE
Локальный запуск без интернета

### DISCUSSION THREAD

**Discussion by N1k0lay** [doka-guide/platform#1265] [Q&A] [answered]
Как запустить платформу с контентом локально и без интернета? 

Отключаю интернет, запускаю npm start, при запуске после создания симлинков появляется ошибка, так как идет запрос к CHANGELOG на гитхаб.

**Comment by furtivite** [comment, marked as answer]
Спасибо, что обратил внимание. Да, ошибка есть, я перенёс её в [issue](https://github.com/doka-guide/platform/issues/1266). Мы обязательно решим.

**Reply by furtivite** [reply]
@N1k0lay не очень правильно было бы рекомендовать локальное решение, но если сборка без интернета нужна здесь и сейчас, то описал временное решение [тут](https://github.com/doka-guide/platform/issues/1266#issuecomment-2112612048).
